bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_handler.c

```diff
@@ -180,13 +180,8 @@ pubsub_tcpHandler_t *pubsub_tcpHandler_create(pubsub_protocol_service_t *protoco
 //
 void pubsub_tcpHandler_destroy(pubsub_tcpHandler_t *handle) {
     if (handle != NULL) {
-        celixThreadRwlock_readLock(&handle->dbLock);
-        bool running = handle->running;
-        celixThreadRwlock_unlock(&handle->dbLock);
-        if (running) {
-            celixThreadRwlock_writeLock(&handle->dbLock);
-            handle->running = false;
-            celixThreadRwlock_unlock(&handle->dbLock);
+        if (__atomic_load_n(&handle->running, __ATOMIC_ACQUIRE)) {
+            __atomic_store_n(&handle->running, false, __ATOMIC_RELEASE);
```

Comment:
This always writes false to running? Previously there was a check on running. I think this is fine, just curious.

Reply:
Hmm? The check is still there?

Reply:
Must be sleep deprivation.. ignore it..